Description: Fix indentation of uwsgidecorators.py
 Python 3 throws 'TabError: inconsistent use of tabs and spaces in indentation'
Author: Leonid Borisenko <leo.borisenko@gmail.com>
Forwarded: no
Last-Update: 2011-07-26

Index: uwsgi/uwsgidecorators.py
===================================================================
--- uwsgi.orig/uwsgidecorators.py	2011-07-26 16:03:52.000000000 +0300
+++ uwsgi/uwsgidecorators.py	2011-07-26 16:04:50.000000000 +0300
@@ -51,7 +51,7 @@
         self.f = f
         spooler_functions[f.__name__] = self.f
         self.f.spool = self.spool
-	self.base_dict = {'ud_spool_func':self.f.__name__}
+        self.base_dict = {'ud_spool_func':self.f.__name__}
 
 class spoolforever(spool):
 
@@ -88,7 +88,7 @@
 
     def __init__(self, num, **kwargs):
         self.num = num
-	self.target = kwargs.get('target', '')
+        self.target = kwargs.get('target', '')
 
     def __call__(self, f):
         uwsgi.register_signal(self.num, self.target, f)
@@ -99,7 +99,7 @@
     def __init__(self, secs, **kwargs):
         self.num = kwargs.get('signum', get_free_signal())
         self.secs = secs
-	self.target = kwargs.get('target', '')
+        self.target = kwargs.get('target', '')
 
     def __call__(self, f):
         uwsgi.register_signal(self.num, self.target, f)
@@ -115,7 +115,7 @@
         self.day = day
         self.month = month
         self.dayweek = dayweek
-	self.target = kwargs.get('target', '')
+        self.target = kwargs.get('target', '')
 
     def __call__(self, f):
         uwsgi.register_signal(self.num, self.target, f)
@@ -129,7 +129,7 @@
     def __init__(self, secs, **kwargs):
         self.num = kwargs.get('signum', get_free_signal())
         self.secs = secs
-	self.target = kwargs.get('target', '')
+        self.target = kwargs.get('target', '')
 
     def __call__(self, f):
         uwsgi.register_signal(self.num, self.target, f)
@@ -141,7 +141,7 @@
     def __init__(self, fsobj, **kwargs):
         self.num = kwargs.get('signum', get_free_signal())
         self.fsobj = fsobj
-	self.target = kwargs.get('target', '')
+        self.target = kwargs.get('target', '')
 
     def __call__(self, f):
         uwsgi.register_signal(self.num, self.target, f)
@@ -170,4 +170,3 @@
         t.start()
         return self.f
 
-    
